runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingsConfig.java

```diff
@@ -40,6 +40,17 @@
 )
 public interface ImplingsConfig extends Config
 {
+	@ConfigItem(
+			position = 0,
+			keyName = "whiteOnly",
+			name = "Only White Minimap Labeling",
+			description = "Configures minimap impling tags to be displayed in white only"
+	)
+	default boolean minimapWhiteOnly()
```

Comment:
How is this useful? just configure all implings to have white color.

Reply:
The feature request was specific to the minimap, not to the minimap AND the tile-overlay, etc. Therefore, the tile-setting is still how it is and offers the ability for it to be white on JUST the minimap labels.